src/components/ProductList/ProductList.jsx

```diff
@@ -0,0 +1,19 @@
+import { Text, FlatList } from "react-native";
+import ProductItem from "../ProductItem/ProductItem";
+
+import styles from "./styles";
+
+import pizzaList from "../../mock/pizzaList";
+
+export default function ProductList() {
+  return pizzaList.length ? (
+    <FlatList
+      data={pizzaList}
+      renderItem={({ item }) => <ProductItem itemData={item} />}
+      keyExtractor={(_, idx) => idx}
```

Comment:
Не рекомендується використовувати індекси для `key` , краще додайте унікальні ІД до ваших моків.

> We don’t recommend using indexes for keys if the order of items may change. This can negatively impact performance and may cause issues with component state. Check out Robin Pokorny’s article for an [in-depth explanation on the negative impacts of using an index as a key](https://robinpokorny.com/blog/index-as-a-key-is-an-anti-pattern/). If you choose not to assign an explicit key to list items then React will default to using indexes as keys.

[keys](https://legacy.reactjs.org/docs/lists-and-keys.html#keys)
[lists](https://react.dev/learn/rendering-lists)